rustls/src/crypto/mod.rs

```diff
@@ -455,6 +477,18 @@ pub trait ActiveKeyExchange: Send + Sync {
     fn group(&self) -> NamedGroup;
 }
 
+/// The result from [`SupportedKxGroup::start_and_complete()`].
+pub struct CompletedKeyExchange {
```

Comment:
I wondered if this should have a `Debug` impl, but it would need to be custom: `SharedSecret` doesn't impl `Debug` (and shouldn't) - so probably not worth it.